### PR TITLE
getting started spoke owners

### PR DESCRIPTION
**Pull request by raghuvanshir**
This PR adds the getting started documentation for the Spoke owners. This content is for the non technical folks. there will be another page for the technical spoke team

### CODE REVIEW

File: source/documentation/getting_started/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+| Mojo-Azure-SPOKE_MonitoringContributor  | Monitoring Contributor         |                                | Monitoring contributor with the ability to manage monitoring settings and data in the spoke.          |
+| Mojo-Azure-SPOKE_StandardAdministrator  |                                | StandardAdministrator          | Standard administrator with custom role-based access control in the spoke.                            |
+| Mojo-Azure-SPOKE_PrivilegedAdministrator|                                | PrivilegedAdministrator        | Privileged administrator with custom role-based access control in the spoke.                          |
+
+> Note : The first group in the table is Mojo-Azure-SPOKE_Owner of which you are part of. You should `not add` any accounts to this group.
+
+2. You as spoke owner will be the owner of these Azure AD groups. 
+
+3. Once you have determine the appropriate roles and permissions required for your team members in your spoke use the azure portal to add the user accounts to the respective Azure AD group.
```
Comment (Taylorstu) [inline source/documentation/getting_started/index.html.md.erb:73]: use determined

File: source/documentation/getting_started/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+5. Click on the "+ Add" button to add a new role assignment.
+
+6. In the "Add role assignment" panel, select the "Owner" role from the "Role" drop-down menu.
+
+7. Under "Members" tab, Specify the ALZ devops service principal. In prod this is `MoJO-PROD-LandingZone-Elevated`, likewise for Devl it is `MoJO-Devl-LandingZone-Elevated`
+
+8. Click on the "Save" button to create the role assignment.
+
+9. The "Owner" role will now be granted to the specified sservice principal. We can now start spinning up the core resources.
```
Comment (Taylorstu) [inline source/documentation/getting_started/index.html.md.erb:46]: should service

File: source/documentation/getting_started/index.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,123 @@
+---
+title: Getting Started Spoke Owners
+last_reviewed_on: 2023-05-19
+review_in: 6 months
+weight: 4000
+---
+
+# Getting Started Spoke Owners
+
+As a spoke owner in the Azure Landing Zone (ALZ), you have the ability to perform self-service actions such as creating subscriptions, managing access, and utilizing available resources to fulfill your responsibilities effectively. This guide aims to provide step-by-step instructions and resources to enable spoke owners to independently manage their subscriptions and access permissions.
```
Comment (Taylorstu) [inline source/documentation/getting_started/index.html.md.erb:10]: fulfil

**Review by Taylorstu — CHANGES_REQUESTED** [review]
I've left some comments. Minor Grammar but looks good Ravi

File: source/documentation/getting_started/index.html.md.erb (hunk 2 above)
Comment (raghuvanshir) [reply source/documentation/getting_started/index.html.md.erb:46]: fixed

**Review by Taylorstu — APPROVED** [review]
Looks Good

**Pull request merged**